data/strings/strings.txt

```diff
@@ -22904,6 +22904,141 @@
     zh-Hans = 加载地图
     zh-Hant = 載入地圖
 
+  [pref_tts_speedcams_auto]
+    tags = android
```

Comment:
@gpesquero can you please add a `comment =` tag with a bit more details about the context for translators?

Reply:
> @gpesquero can you please add a `comment =` tag with a bit more details about the context for translators?

Done!